src/lib/util/ModUtil.php

```diff
@@ -11,7 +11,7 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\ControllerNameParser;
 use Symfony\Bundle\FrameworkBundle\Controller\ControllerResolver;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Zikula\Core\Event\GenericEvent;
```

Comment:
changes to ModUtil (or any Util class) in the master branch are a waste of time. They are **all** going to be deleted.